aeon/distances/tests/test_pairwise.py

```diff
@@ -218,58 +226,58 @@ def test_new_single_to_multiple_distances(dist):
         )
 
     _validate_single_to_multiple_result(
-        _create_test_distance_numpy(5),
-        _create_test_distance_numpy(3, 1, 5, random_state=2),
+        make_series(5, 1, return_numpy=True, random_state=1),
+        make_example_3d_numpy(3, 1, 5, random_state=2)[0],
         dist["name"],
         dist["distance"],
         dist["pairwise_distance"],
     )
 
     _validate_single_to_multiple_result(
-        _create_test_distance_numpy(3, 1, 5)[0],
-        _create_test_distance_numpy(5, 1, 5, random_state=2),
+        make_series(1, 5, return_numpy=True, random_state=1),
+        make_example_3d_numpy(5, 1, 5, random_state=2)[0],
         dist["name"],
         dist["distance"],
         dist["pairwise_distance"],
     )
 
     # Multivariate tests
     _validate_single_to_multiple_result(
-        _create_test_distance_numpy(5, 5),
-        _create_test_distance_numpy(5, 5, 5, random_state=2),
+        make_series(5, 5, return_numpy=True, random_state=1),
+        make_example_3d_numpy(5, 5, 5, random_state=2)[0],
         dist["name"],
         dist["distance"],
         dist["pairwise_distance"],
     )
 
     # Unequal length tests
     _validate_single_to_multiple_result(
-        _create_test_distance_numpy(3),
-        _create_test_distance_numpy(3, 1, 5, random_state=2),
+        make_series(3, return_numpy=True, random_state=1),
+        make_example_3d_numpy(3, 1, 5, random_state=2)[0],
         dist["name"],
         dist["distance"],
         dist["pairwise_distance"],
     )
 
     _validate_single_to_multiple_result(
-        _create_test_distance_numpy(5),
-        _create_test_distance_numpy(3, 1, 3, random_state=2),
+        make_series(3, return_numpy=True, random_state=1),
+        make_example_3d_numpy(3, 1, 3, random_state=2)[0],
         dist["name"],
         dist["distance"],
         dist["pairwise_distance"],
     )
 
     _validate_single_to_multiple_result(
-        _create_test_distance_numpy(5, 3),
-        _create_test_distance_numpy(5, 5, 5, random_state=2),
+        make_series(5, 3, return_numpy=True, random_state=1),
```

Comment:
To make it an unequal-length test:
```suggestion
        make_series(3, 5, return_numpy=True, random_state=1),
```